app/views/foreman_puppet/common/_puppetclasses_or_envs_changed.html.erb

```diff
@@ -1,4 +1,9 @@
 <% title _("Changed environments") %>
+<%= breadcrumbs(:switchable => false,
+                :items => [
+                  { :caption => _('Puppet Environments'), :url => environments_path },
```

Comment:
Nit: Replace `:key =>` with `key:`

Reply:
If you're changing the new hash syntax, you should also fix it in here:
https://github.com/theforeman/foreman_puppet/commit/8ae23e3cb064c89dcd82c6f1eeeaca588d5fe581

Reply:
Actually, we might want to add that to rubocop check

Reply:
updated with the new style + fixed in the welcome page.

Reply:
Thanks! Maybe you can either rename the second commit or squash them

Reply:
I think you can squash and merge them together when merging the PR